.github/workflows/transport-amqp.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+name: transport-amqp
+on:
+  push:
+    paths:
+      - 'component/didcomm/transport/amqp/**'
+  pull_request:
+    paths:
+      - 'component/didcomm/transport/amqp/**'
+jobs:
+  linter:
+    name: Go linter
+    timeout-minutes: 10
+    env:
+      LINT_PATH: component/didcomm/transport/amqp
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checks linter
+        timeout-minutes: 10
+        run: make lint
+  unitTest:
+    name: Unit test
+    runs-on: ubuntu-18.04
+    services:
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 5672:5672
+    timeout-minutes: 15
+    env:
+      UNIT_TESTS_PATH: component/didcomm/transport/amqp
+      RABBITMQ_HOST: localhost
+    steps:
+      - name: Setup Go 1.15
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+        id: go
+
+      - uses: actions/checkout@v2
+
+      - name: Run unit test
+        timeout-minutes: 15
+        run: make unit-test
+
+      - name: Upload coverage to Codecov
```

Comment:
It seems that we didn't get a codecov comment for this PR.

Reply:
Perhaps a rebase will help.

Reply:
Looks like rebase did the trick, we have coverage